docker-compose/Makefile

```diff
@@ -1,4 +1,4 @@
-DOCKER_HOST_IP ?= $(shell echo ${DOCKER_HOST} | grep -o "[0-9]\{1,3\}\.[0-9]\{1,3\}\.[0-9]\{1,3\}\.[0-9]\{1,3\}" || echo localhost)
+DOCKER_HOST_IP ?= $(shell echo ${DOCKER_HOST} | grep -o "[0-9]\{1,3\}\.[0-9]\{1,3\}\.[0-9]\{1,3\}\.[0-9]\{1,3\}" || hostname)
```

Comment:
👍  `hostname` is better than localhost.

In some cases it won't be resolved from within the container. Should we document that the developer needs to ensure the hostname is resolvable ?

Reply:
Yes.